src/Auth/AuthContainer.js

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+// import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import Header from 'grommet/components/Header';
```

Comment:
This file is missing `` import { PropTypes } from 'react'; `` which is used on https://github.com/esbb48/Kizuna/pull/4/files#diff-46ed9dd889d764dc44913c7bf73559a1R19